### ISSUE TITLE
Make `SORT_ORDER` to be a switcher

### ISSUE THREAD

**Issue by vrom911** [JackKiefer/herms#31]
For the moment if you want to get the recipes grouped by tag you need to input 
```
herms list  -s tags   OR   herms list -s default
```
The problem is that by default the ordering is `default`, so I don't see any sense in having option with `-s default`. Also writing `tags` or `default` by hands not really user-friendly (I couldn't manage to find out this magic words without going deeper into the code sources). 

What I suggest is to have a switch so it will turn on grouping by tags like this
```
herms list --sort
```
Also short `-s` is not the best choice because there is already `s` for servings, I thing it could make users confused..

But all of this is just my opinion as a person who used `herms`. I just want to help in making this tool even better! :sparkles:

**Comment by JackKiefer** [comment]
I think these are some solid ideas! I'd definitely welcome a pull request that would implement this. Perhaps something like `-r` would be a good option for a short flag. 

I'll throw on some labels!

**Comment by vrom911** [comment]
Excellent! Then I'm taking this one. Thank you very much for your comments on this :blush:

**Comment by JackKiefer** [comment]
Implemented by #32
